src/components/QuickOpenModal.js

```diff
@@ -357,8 +357,20 @@ export class QuickOpenModal extends Component<Props, State> {
     return !this.getResultCount() && !!query;
   }
 
+  renderLoading = () => {
+    const { symbols, selectedSource } = this.props;
+
+    if (selectedSource) {
```

Comment:
we should still check to see if we're looking for a function or variable?

maybe `isFunctionQuery() || isVariableQuery()`